pydataapi/pydataapi.py

```diff
@@ -356,8 +366,12 @@ def __init__(
         transaction_id: Optional[str] = None,
         client: Optional[boto3.session.Session.client] = None,
         rollback_exception: Optional[Type[Exception]] = None,
+        rds_client: Optional[boto3.session.Session.client] = None,
```

Comment:
Should we accept access_key_id, secret_access_key, region and endpoint?

Reply:
Hm. We could. Although, that might be a bigger change. I think it is safe to assume most systimes have that information in `~/.aws`

Reply:
OK, I don't touch this argument. Users may want to set a client object.